src/engine/SCons/Util.py

```diff
@@ -34,15 +34,12 @@
 import codecs
 import pprint
 import hashlib
+from collections import UserDict, UserList, UserString, OrderedDict
+from collections.abc import MappingView
 
 PY3 = sys.version_info[0] == 3
```

Comment:
this line isn't used any longer

Reply:
gotcha.  I'll push change after CI builds finish.